frontend/shared/components/src/icons/DateBox.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+    <div class="date-image-box">
+        <div class="overlay">
+            <span v-if="date" class="day">{{ Formatter.day(date) }}</span>
+            <span v-if="date" class="month">{{ Formatter.capitalizeFirstLetter(Formatter.month(date)) }}</span>
+            <span v-if="!date" class="month">{{ $t('Geen datum') }}</span>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { Formatter } from '@stamhoofd/utility';
+
+defineProps<{
+    date?: Date;
+}>();
```

Comment:
```suggestion
withDefaults(defineProps<{
    date?: Date | null;
}>(), {date: null});
```
Anders zal vue warnings geven in de console